ShowBuildMenu.bat

```diff
@@ -18,9 +18,10 @@ echo C. (Step 2) Activate a test configuration.
 echo D. (Step 3) Run tests using active configuration.
 echo.
 echo --- BUILD ---
-echo E. Build NHibernate (Debug)
-echo F. Build NHibernate (Release)
-echo G. Build Release Package (Also runs tests and creates documentation)
+echo E. Generate async code ((re)generates files in Async sub-folders)
```

Comment:
You now have done it, but dropping "Contributor Guide.html" by the way and not changing the letter I have put in markdown version.
Was the drop voluntary?